src/views/Start/Start.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useCallback, useEffect, useState } from 'react'
+
+import {
+  Box,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardIcon,
+  Container,
+  Separator,
+  Spacer,
+} from 'react-neu'
+import styled from 'styled-components'
+
+import StartPage from 'components/StartPage'
+import StartPageHeader from 'components/StartPageHeader'
+import StartSplit from 'components/StartSplit'
+
+import CoinDeskIcon from 'components/icons/CoinDeskIcon'
+import CoinTelegraphIcon from 'components/icons/CoinTelegraphIcon'
+import ForbesIcon from 'components/icons/ForbesIcon'
+import GizmodoIcon from 'components/icons/Gizmodo'
+import TheDefiantIcon from 'components/icons/TheDefiantIcon'
+import TheRegisterIcon from 'components/icons/TheRegisterIcon'
+
+const ASTRONAUTS = [
+  '👨‍🚀',
+  '👨🏻‍🚀',
+  '👨🏼‍🚀',
+  '👨🏽‍🚀',
+  '👨🏾‍🚀',
+  '👩‍🚀',
+  '👩🏻‍🚀',
+  '👩🏼‍🚀',
+  '👩🏽‍🚀',
+  '👩🏾‍🚀‍',
+  '👩🏿‍🚀'
+]
+
+const Start: React.FC = () => {
+  const [astronaut, setAstronaut] = useState('👨‍🚀')
+
+  const updateAstronaut = useCallback(() => {
+    const newAstro = ASTRONAUTS[Math.floor(Math.random()*ASTRONAUTS.length)]
+    setAstronaut(newAstro)
+  }, [setAstronaut])
+
+  useEffect(() => {
+    const refresh = setInterval(updateAstronaut, 1000)
+    return () => clearInterval(refresh)
+  }, [updateAstronaut])
+
+  return (
+    <StartPage>
+      <StyledHero>
+        <StartPageHeader
+          icon={`${astronaut}❤️🍠`}
+          subtitle="Fair launch, open participation, and inclusive community."
+          title="Fair finance by the people, for the people."
+        />
+        <Container size="lg">
+          <Box row justifyContent="center">
+            <Button text="Open App" href="/" />
+            <Spacer />
+            <Button text="Join the Community" variant="secondary" />
+          </Box>
+        </Container>
+      </StyledHero>
+      <Container size="lg">
+        <Spacer size="lg" />
+        <Separator />
+        <Spacer size="lg" />
+        <StyledSectionIcon>⚖️</StyledSectionIcon>
+        <Spacer size="lg" />
+        <StyledSectionTitle>Fair finance for everyone.</StyledSectionTitle>
+        <StyledSectionDescription>Yam is owned and controlled by our community of Yam token holders.</StyledSectionDescription>
+        <Spacer size="lg" />
+        <StartSplit>
+          <Card>
+            <CardIcon>💸</CardIcon>
+            <CardContent>
+              <StyledCardName>Growing treasury</StyledCardName>
+              <Spacer size="sm" />
+              <StyledCardDescription>The YAM treasury is vibrant and growing fast.</StyledCardDescription>
+            </CardContent>
+            <CardActions>
+              <Box row justifyContent="center">
+                <Button text="View treasury" variant="secondary" />
```

Comment:
buttons are not linked to anything, we need to link them to the relevant pages
`/dashboard` for the moment